.circleci/config.yml

```diff
@@ -58,4 +58,4 @@ jobs:
       # run linter
       - run: yarn lint-ts
       # run tests!
-      - run: yarn test --verbose --coverage=true --forceExit --detectOpenHandles
\ No newline at end of file
+      - run: yarn test --runInBand --verbose --coverage=true --forceExit --detectOpenHandles
```

Comment:
What is the rationale for doing this?

Reply:
Well, a rudimentary explanation goes like this:
Apparently, jest hits some performance issues on CD/CI server environments, Why?, I don't know yet. This causes circle ci builds to timeout with no output. This flag seems to fix the problem by "Running the tests sequentially ".

This is not much to ride on, and I will have a full explanation by the time am done resolving this issue

Reply:
Nice work on finding this.  Can you move this commit to a different PR that we can merge ASAP so that we unblock the other PRs stuck with broken tests?